src/Compilers/CSharp/Portable/Parser/Lexer.cs

```diff
@@ -770,7 +770,7 @@ private void ScanSyntaxToken(ref TokenInfo info)
                     }
                     else if (TextWindow.PeekChar(1) == '$' && TextWindow.PeekChar(2) == '"')
                     {
-                        this.ScanInterpolatedStringLiteral(isVerbatim: true, ref info);
```

Comment:
stopped passing data along.  this requires teh caller to figure this info out (which gets much more complex with raw-string literals.  now, all the determination code is centralized in a single location.